packages/components/src/date-time/date.js

```diff
@@ -13,6 +13,7 @@ import { Component } from '@wordpress/element';
  * Module Constants
  */
 const TIMEZONELESS_FORMAT = 'YYYY-MM-DDTHH:mm:ss';
+const isRTL = () => document.documentElement.dir === 'rtl';
```

Comment:
This was also used in https://github.com/WordPress/gutenberg/commit/f91e66ca2bcd9e1e7b97aa39c01f2888fc751712. I think it's fine to use it here too instead of `getEditorSettings().isRTL`.

cc @youknowriad

Reply:
Yes, so the components package shouldn't depend on the editor because these are generic components that can be used anywhere. 

So this approach is a workaround, It's ok.
A better alternative here for me is to have an `isRtl` prop for this component and pass the correct value  from the editor module using `getEditorSettings().isRTL`.  (In the Popover component it was harder to achieve because it's used in too many places).

That said I'm still ok with this.